src/components/Layout/Layout.tsx

```diff
@@ -19,54 +20,35 @@ import EDSBanner from "../EDSBanner"
 import RCSubNav from "../RCSubNav/RCSubNav"
 
 interface LayoutProps {
-  sidebar?: ReactElement
   activePage?: RCPage
-  sidebarPosition?: "right" | "left"
   isAuthenticated?: boolean
   searchAggregations?: Aggregation[]
   searchResultsCount?: number
   bannerNotification?: string
 }
 
 /**
- * The Layout component wraps the TemplateAppContainer from the DS and
+ * The Layout component wraps the Template from the DS and
  * controls the rendering of Research Catalog header components per-page.
  */
 const Layout = ({
   searchAggregations,
   children,
   isAuthenticated,
-  sidebar,
   activePage,
   searchResultsCount,
-  sidebarPosition = "left",
   bannerNotification,
 }: PropsWithChildren<LayoutProps>) => {
   const showSearch = activePage === "search"
   const showNotification = activePage === "" || activePage === "search"
   return (
     <DSProvider>
-      <TemplateAppContainer
-        // This is a workaround to fix a text-wrapping issue when page is zoomed in on
-        // TODO: Address this issue in the DS
-        sx={{
-          "main > div": { maxWidth: "100vw" },
-          rowGap: {
-            base: "grid.m",
-            md: "grid.l",
-          },
-          main: {
-            rowGap: {
-              base: "grid.m",
-              md: "grid.l",
-            },
-          },
-        }}
-        breakout={
+      <Template variant="full">
```

Comment:
Did `TemplateSidebar` not work for you?

Reply:
Understood. This is now a prime opportunity to implement the [recommendations](https://nypl-design-system-git-reservoir-v4-nypl.vercel.app/?path=/docs/style-guide-responsive-grid--docs) for a responsive grid/column layout.